src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -0,0 +1,424 @@
+package org.mifos.integrationtest.cucumber.stepdef;
+
+import com.github.tomakehurst.wiremock.stubbing.ServeEvent;
+import io.cucumber.core.internal.com.fasterxml.jackson.databind.JsonNode;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.RestAssured;
+import io.restassured.builder.ResponseSpecBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+import java.util.Random;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.google.common.truth.Truth.assertThat;
+import static org.mifos.integrationtest.common.HttpMethod.PUT;
+
+public class VoucherManagementStepDef extends BaseStepDef{
+    private static String requstId;
+    private static String createVoucherBody;
+    private static String activateVoucherBody;
+    private static String redeemVoucherBody;
+    private static String redeemVoucherResponseBody;
+    private static  String callbackBody;
+    private static String serialNumber;
+    private static String voucherNumber;
+    private static String cancelVoucherBody;
+    private static String suspendVoucherBody;
+    private static String registeringInstitutionId = "SocialWelfare";
+    @Autowired
+    MockServerStepDef mockServerStepDef;
+
+
+    @Given("I can create an VoucherRequestDTO for voucher creation")
+    public void iCreateAnIdentityMapperDTOForRegisterBeneficiary() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("{\n");
```

Comment:
Though I accept this here, these kinds of JSON structures will be difficult to manage.